arcana/core/utils.py

```diff
@@ -16,11 +16,15 @@
 from collections.abc import Iterable
 import logging
 import attr
-from arcana._version import __version__
 from pydra.engine.task import FunctionTask
 from pydra.engine.specs import BaseSpec, SpecInfo
 from arcana.exceptions import ArcanaUsageError, ArcanaNameError, ArcanaVersionError
 
+# Avoid arcana.__version__ causing a circular import
```

Comment:
Why is this necessary? Where does the circular import come from?

Reply:
If the original import was also changed to
```python
from arcana import __version__
```
it would result in a circular import

Reply:
ok, I see